src/conf.py

```diff
@@ -234,6 +250,21 @@ def setup(app):
 """
 
 # At bottom of every document
+doc_pdf = """
```

Comment:
This message should just read:
Download this documentation as a PDF.
With the entire sentence being the link anchor text.